UPGRADE-2.0.md

```diff
@@ -4,3 +4,9 @@
 
 Since it was moved to [datagrid-bundle](https://github.com/fsi-open/datagrid-bundle),
 you will need to install it in order to be able to use the extension.
+
+## Use null in boolean column only when you mean it
+
+If any of your boolean columns can contain `null` then empty string will be displayed
```

Comment:
``
If any of your boolean columns can contain `null`, then an empty string will be displayed
instead of the value passed as `false_value`. You should either ensure that the column does not
contain `null` or that it adheres to the new behavior.
``